### PR TITLE
Update go to 1.19.3

### PR DESCRIPTION
**Pull request by luhring**
Resolves CVE-2022-41716

### CODE REVIEW

File: go.yaml [hunk 1]
```diff
@@ -15,8 +15,16 @@ package:
       - build-base
       - bash
 
+secfixes:
+  1.19.3-r0:
+    - CVE-2022-41716
+
 environment:
   contents:
+    repositories:
```
Comment (kaniini) [inline go.yaml:24]: This isn't needed and shouldn't be present.  Instead, use `gsutil -m rsync` to clone the repository locally.  I can add some helpers to the Makefile to help with this.

**Review by kaniini — CHANGES_REQUESTED** [review]

**Review by kaniini — APPROVED** [review]

**Comment by tuananh** [comment]
@luhring off topic question but may i ask why is epoch set to 1?

**Comment by kaniini** [comment]
Because go binaries are staticly linked, we have to rebuild them when Go receives security updates, so we increase the epoch to trigger a rebuild.

**Comment by tuananh** [comment]
that seems like a bit of manual, error-prone process, maybe it's better to have a check in CI for if a package like `go` get updated? which print out error message like "trivy is staticly linked with go, because this PR updates go so you need to bump epoch for trivy too"

**Comment by kaniini** [comment]
We use `dag` to handle this.  https://github.com/wolfi-dev/dag

**Comment by tuananh** [comment]
This is nice. I saw there's a PR for it but it's closed? https://github.com/wolfi-dev/os/pull/19

**Comment by imjasonh** [comment]
That work moved to https://github.com/wolfi-dev/dag -- there are a few goals of that tool:

1. visualization of the dependency graph, just to be able to tell what depends on what, track over time, etc.
2. execution of the graph when things change -- e.g., when `go` is rebuilt, know to update and rebuild `trivy`, etc.
3. execution of the graph remotely -- instead of having the build graph manually maintained in `Makefile`, have `dag` create a K8s pod that runs the necessary builds in order on cloud VMs. This will also help us support arm64 builds better.

If this is interesting feel free to take it for a spin, file issues, and ask questions. We can discuss in more detail in a future public community call as well.

**Pull request merged**